scripts/prepublish.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const regex = /module\.exports = punycode;/;
+
+var sourceContents = fs.readFileSync(path.resolve(__dirname, '../punycode.js'), 'utf-8');
+
+if (!regex.test(sourceContents)) {
```

Comment:
I thought the use of `includes` would be fine (same as `const` etc.) since this script would only ever get run in a modern Node.js version.

But then I remembered… The `prepublish` script is run on `npm install` as well. Still, it doesn’t matter, since the HEAD version of Punycode.js supports recent versions of Node.js only.